### DISCUSSION TITLE
结果输出的建议

### DISCUSSION THREAD

**Discussion by piaoliuchong** [Guovin/iptv-api#771] [General]
"偏好设置"选择了ipv4 ipv6数量，那么输出结果“接口协议类型”选择项为"ipv4  ipv6  全部"。能不能改为分别输出ipv4  ipv6两个结果文件？
目前操作是运行两边。第一轮ipv4，

第二次操作ipv6，在选择了“使用历史结果，使用离线数据后”，又要重新载入一次订阅源列表，好像选择历史结果没生效。

建议将"历史结果"改为选择项，每次运行生成缓存文件，通过选择不同的缓存文件进行再次筛选。
"离线文件"功能没发现怎么用。

**Comment by Guovin** [comment]
1.目前没有按ipv类型输出两个结果文件的计划，这会影响复杂度。
2.历史结果是基于上一次运行生成的可用结果。离线文件目前只有组播和酒店，如果你想要订阅源的离线数据即缓存，这跟 #769 是一样的需求。
3.目前是有默认生成结果缓存文件的，即output/cache.pkl。
4.提醒：我发现你提了多个相同类似的讨论，请注意，最好总结成一个，否则频繁的邮件提醒会打扰大家，谢谢。
